verify_phpunit_xml/verify_phpunit_xml.sh

```diff
@@ -44,36 +44,18 @@ existingtests=$(cd ${gitdir} && find . -name tests | sed 's/^\.\/\(.*\)$/\1/g')
 ignoretests=""
 
 # Unit test classes to look for with each file (must be 1 and only 1). MDLSITE-2096
-# TODO: Some day replace this with the list of abstract classes, using some classmap going up to phpunit top class.
+# TODO: When we stop supporting these class names, we can remove them.
 unittestclasses="
-    advanced_testcase
     area_test_base
     badgeslib_test
-    base_testcase
-    basic_testcase
     cachestore_tests
-    core_backup_backup_restore_base_testcase
-    core_reportbuilder_testcase
     data_loading_method_test_base
-    data_privacy_testcase
-    database_driver_testcase
-    externallib_advanced_testcase
-    googledocs_content_testcase
-    grade_base_testcase
-    lti_advantage_testcase
     manage_category_test_base
     messagelib_test
```

Comment:
It looks like we can remove the following, and the test will still pass:
- `messagelib_test`
- `mod_quiz\\\\attempt_walkthrough_from_csv_test`

Is there a reason they were not removed here?